### PR TITLE
Add methods to retrieve the Local and Global Context

### PR DESCRIPTION
**Pull request by bordoni**
_Ref:_ [C#126089](http://central.tri.be/issues/126089)
---

### CODE REVIEW

File: src/Tribe/Template.php [hunk 1]
```diff
@@ -172,7 +172,33 @@ public function set_template_context_extract( $value = false ) {
 	}
 
 	/**
-	 * Sets a Index inside of the global or local context
+	 * Gets the local template context
+	 *
+	 * @since  TBD
+	 *
+	 * @todo  Apply filters here for local context
```
Comment (sc0ttkclark) [inline src/Tribe/Template.php]: @bordoni Do you need filters added?

File: src/Tribe/Template.php [hunk 2]
```diff
@@ -172,7 +172,33 @@ public function set_template_context_extract( $value = false ) {
...
+	public function get_template_context() {
+		return $this->context;
+	}
+
+	/**
+	 * Gets the global template context
+	 *
+	 * @since  TBD
+	 *
+	 * @todo  Apply filters here for global context
```
Comment (sc0ttkclark) [inline src/Tribe/Template.php]: @bordoni Do you need filters added?

Comment (bordoni) [reply src/Tribe/Template.php]: Not now, but I wanted to make sure we could do that for template filtering of local and global context from customers.

**Comment by sc0ttkclark** [comment]
Needs changelog, or no?

**Comment by bordoni** [comment]
Closing in favor of @lucatume code on #1031

**Pull request closed without merging**